.github/workflows/test.yml

```diff
@@ -0,0 +1,71 @@
+name: Test
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+jobs:
+  test-web:
```

Comment:
The Action is already named `Test`, so we could shorten the Jobs to just be named after the context like `Web`, `Android` and `iOS`.